packages/core/test/unit/initializer/npm-registry.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { getRegistry } from '../../../src/initializer/npm-registry.js';
+import { expect } from 'chai';
+import { testInjector } from '@stryker-mutator/test-helpers';
+import sinon from 'sinon';
+import { execaCommandSync } from 'execa';
+
+const DEFAULT_REGISTRY = 'https://registry.npmjs.com';
+
+describe('npm registry', () => {
+  describe('get registry', () => {
+    let oldNpmConfigRegistry: string | undefined;
+    let oldNpmCommand: string | undefined;
+
+    before(() => {
```

Comment:
This should be `beforeEach` instead of `before` (or change `afterEach` to be `after`)